exporter/sumologicexporter/sender.go

```diff
@@ -119,8 +123,12 @@ func (s *sender) send(ctx context.Context, pipeline PipelineType, body io.Reader
 		req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
 		req.Header.Add("X-Sumo-Fields", flds.string())
 	case MetricsPipeline:
-		// ToDo: Implement metrics pipeline
-		return errors.New("current sender version doesn't support metrics")
+		switch s.config.MetricFormat {
```

Comment:
Wouldn't it be better to check this in `newSender` and then fail fast when we received a metrics format that we do not support?

Reply:
We check it in exporter